projects/ngneat/error-tailor/src/lib/control-error.directive.spec.ts

```diff
@@ -144,6 +149,18 @@ describe('ControlErrorDirective', () => {
       expect(spectator.query(byText('required error'))).toBeFalsy();
     });
 
+    it('should show errors on programmatic access', () => {
+      const explicitInput = spectator.query<HTMLInputElement>(byPlaceholder('Explicit'));
```

Comment:
The test doesn't test your new functionality. Call showError and hideError and see the errors is visible.